lms/djangoapps/ora_staff_grader/views.py

```diff
@@ -84,3 +84,59 @@ def get_rubric_config(self, request, usage_id):
             'target_rubric_block_id': usage_id
         }
         return call_xblock_json_handler(request, usage_id, 'get_rubric', data)
+
+
+class SubmissionFetchView(RetrieveAPIView):
+    """
+    GET submission contents and assessment info, if any
+
+    Response: {
+        gradeData: {
+            score: (dict or None) {
+                pointsEarned: (int) earned points
+                pointsPossible: (int) possible points
+            }
+            overallFeedback: (string) overall feedback
+            criteria: (list of dict) [{
+                name: (str) name of criterion
+                feedback: (str) feedback for criterion
+                selectedOption: (str) name of selected option or None if feedback-only criterion
+            }]
+        }
+        response: {
+            text: (list of string), [the html content of text responses]
+            files: (list of dict) [{
+                downloadURL: (string) file download url
+                description: (string) file description
+                name: (string) filename
+            }]
+        }
+    }
+    """
+    authentication_classes = (
+        JwtAuthentication,
+        BearerAuthenticationAllowInactiveUser,
+        SessionAuthenticationAllowInactiveUser,
+    )
+    permission_classes = (IsAuthenticated,)
```

Comment:
A note that we (collectively) should probably have a base class that standardizes the auth/permission classes